core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -109,7 +113,7 @@ def __init__(
         if lambda_function:
             self._function = lambda_function
         elif code and handler:
-            self._function = LambdaFactory.function(
+            self._function = LambdaFactory.function(  # type: ignore
```

Comment:
Had trouble with `mypy` here with the new optional map `**function_props`

```
core/aws_ddk_core/stages/sqs_lambda.py:116: error: Argument after ** must be a mapping, not "Optional[Dict[str, Any]]"
core/aws_ddk_core/stages/glue_transform.py:104: error: Argument after ** must be a mapping, not "Optional[Dict[str, Any]]"
```

Not sure how to get around it complaining about and optional arg here.